### PR TITLE
(SIMP-5439) BEAKER_fips=yes setup fails on centos/7 vbox

### PR DESCRIPTION
**Pull request by lnemsick-simp**
This is a *quick* patch to get BEAKER_fips=yes functioning again.
If the simp-fips modules is not in the .fixtures.yml file for the component under test,
this patch *assumes* connectivity to PuppetForge.  A more complete
solution that is also appropriate for isolated development
environments should be done in a different ticket.

SIMP-5439 #close

### CODE REVIEW

File: spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  def initialize
+    FileUtils.rm_rf(File.join(fixtures_path, 'modules'))
+  end
+end
+
+require 'yaml'
+require 'tempfile'
+
+@alt_fixtures = '.fips_fixtures.yml'
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb]: @trevor-vaughan why a class variable?

File: spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  ENV['FIXTURES_YML'] = @alt_fixtures
+
+  %x{bundle exec rake spec_prep}
+}
+
+require 'spec_helper_acceptance'
+
+hosts.each do |host|
+  describe 'FIPS pre-installed' do
+    after(:suite) do
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb]: @trevor-vaughan This spewed an warning:
WARNING: `after(:suite)` hooks are only supported on the RSpec configuration object. This `after(:suite)` hook, registered on an example group, will be ignored. Called from /var/opt/enemsick/orep/rubygem-simp-beaker-helpers/spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb:44:in `block (2 levels) in <top (required)>'.
WARNING: `after(:suite)` hooks are only supported on the RSpec configuration object. This `after(:suite)` hook, registered on an example group, will be ignored. Called from /var/opt/enemsick/orep/rubygem-simp-beaker-helpers/spec/acceptance/suites/fips_from_fixtures/00_default_spec.rb:44:in `block (2 levels) in <top (required)>'.

**Comment by trevor-vaughan** [comment]
@lnemsick-simp You were right on both counts. The last update should fix everything.

**Comment by lnemsick-simp** [comment]
Both issues were addressed by  last set of changes.

**Pull request merged**